### PR TITLE
Add Rubika & Soroush+ recipes

### PR DESCRIPTION
**Pull request by xmha97**
#### Pre-flight Checklist

Please ensure you've completed all of the following.

- [X] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I updated the version package [Updating](https://github.com/ferdium/ferdium-recipes/blob/main/docs/updating.md#4-updating-the-version-number)

### CODE REVIEW

**Comment by xmha97** [comment]
Hi @vraravam
"Rubika" and "Soroush+" messengers are widely used in Iran and I want you to add them to this app.
Please merge this PR.
Thanks.

File: recipes/eitaa/package.json [hunk 1]
```diff
@@ -1,9 +1,14 @@
 {
   "id": "eitaa",
   "name": "Eitaa",
-  "version": "1.0.0",
+  "version": "1.0.1",
+  "repository": "https://github.com/xmha97/FerdiumRecipes",
```
Comment (vraravam) [inline recipes/eitaa/package.json:5]: please remove the `repository` key

File: recipes/bale/package.json [hunk 2]
```diff
@@ -1,9 +1,14 @@
 {
   "id": "bale",
   "name": "Bale",
-  "version": "1.0.0",
+  "version": "1.0.1",
+  "repository": "https://github.com/xmha97/FerdiumRecipes",
```
Comment (vraravam) [inline recipes/bale/package.json:5]: please remove the `repository` key

File: recipes/rubika/package.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "id": "rubika",
+  "name": "Rubika",
+  "version": "1.0.0",
+  "repository": "https://github.com/xmha97/FerdiumRecipes",
```
Comment (vraravam) [inline recipes/rubika/package.json:5]: please remove the `repository` key

File: recipes/rubika/package.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  "license": "MIT",
+  "aliases": [
+    "روبیکا"
```
Comment (vraravam) [inline recipes/rubika/package.json:8]: `alias` needs to be in english - pls change.

File: recipes/soroush/package.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{
+  "id": "soroush",
+  "name": "Soroush+",
+  "version": "1.0.0",
+  "repository": "https://github.com/xmha97/FerdiumRecipes",
+  "license": "MIT",
+  "aliases": [
+    "سروش پلاس"
```
Comment (vraravam) [inline recipes/soroush/package.json:8]: alias needs to be in english - please change

File: recipes/soroush/package.json [hunk 6]
```diff
@@ -0,0 +1,14 @@
+{
+  "id": "soroush",
+  "name": "Soroush+",
+  "version": "1.0.0",
+  "repository": "https://github.com/xmha97/FerdiumRecipes",
```
Comment (vraravam) [inline recipes/soroush/package.json:5]: please remove the `repository` key

**Review by vraravam — CHANGES_REQUESTED** [review]
please do not make changes to multiple recipes in the same PR - please create separate PRs (1 per recipe)

**Pull request closed without merging**